### PR TITLE
Add multi table support

### PR DESCRIPTION
**Pull request by tonylixu**
Cassandra nodetool only supports the following snapshot scenario:
1. Snapshot all keyspaces
2. Snapshot one or more keyspaces and all cfs
3. Snapshot a single cf

This PR will allow snapshot a single keyspace and multiple tables of that single keyspace

### CODE REVIEW

**Comment by tonylixu** [comment]
@usmanismail Ready for review. Test on metrics126 and features Cassandra cluster.

**Comment by usmanismail** [comment]
Looks good to me

File: cassandra_snapshotter/agent.py [hunk 1]
```diff
@@ -229,10 +235,16 @@ def create_upload_manifest(
 
             path = os.path.join(*path)
             glob_results = '\n'.join(glob.glob(os.path.join(path)))
-            files.extend([f.strip() for f in glob_results.split("\n")])
+            for f in glob_results.split("\n"):
```
Comment (shannonwhite) [inline cassandra_snapshotter/agent.py]: Can this just be changed to for `f in glob.glob(os.path.join(path))` ?

**Comment by tonylixu** [comment]
@shannonwhite Good catch, removed glob_results

File: cassandra_snapshotter/snapshotting.py [hunk 2]
```diff
@@ -263,21 +264,24 @@ def upload_node_backups(self, snapshot, incremental_backups):
         prefix = '/'.join(snapshot.base_path.split(
             '/') + [self.get_current_node_hostname()])
 
-        manifest_path = '{}/backupmanifest'.format(expanduser('~'))
+        #manifest_path = '{}/backupmanifest'.format(expanduser('~'))
```
Comment (shannonwhite) [inline cassandra_snapshotter/snapshotting.py]: Should not have this commented out unless you have an additional comment explaining why

File: cassandra_snapshotter/agent.py [hunk 3]
```diff
@@ -228,11 +234,19 @@ def create_upload_manifest(
             path += ['*']
 
             path = os.path.join(*path)
-            glob_results = '\n'.join(glob.glob(os.path.join(path)))
-            files.extend([f.strip() for f in glob_results.split("\n")])
+            if len(exclude_tables_list) > 0:
+                for f in glob.glob(os.path.join(path)):
+                # Get the table name
```
Comment (shannonwhite) [inline cassandra_snapshotter/agent.py]: These comments should be indented

**Comment by tonylixu** [comment]
@shannonwhite Fixed and tested

**Pull request merged**